releasenotes/notes/cc-gate-fidelity-bde7a01dc8c56e29.yaml

```diff
@@ -0,0 +1,24 @@
+---
+upgrade:
```

Comment:
You need a feature note about the new argument on the commutative pass constructors.

Reply:
in my mind it was not API documented 😛  I'll add it!